src/style.css

```
@@ -0,0 +1,153 @@
+html,
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+  box-sizing: border-box;
+}
+
+.card {
+
+  width: 198px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card-image {
+  width: 160px;
+  height: 134px;
+  padding: 32px 19px 36px ;
+  align-self: center;
+}
+
+.card-product__description {
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  margin: 0 16px;
+  color: #060b35;
+}
+
+.card-product__number {
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 10px;
+  line-height: 14px;
+  margin-left: 16px;
+  margin-top: 4px;
+  margin-bottom: 16px;
+  color: #616070;
+
+}
```

группируй модификаторы и разделяй пустой строкой
удали лишнюю пустую строку (по всему коду тоже)
```suggestion
.card-product__number {
  margin-left: 16px;
  margin-top: 4px;
  margin-bottom: 16px;
  
  font-family: Roboto, sans-serif;
  font-style: normal;
  font-weight: normal;
  font-size: 10px;
  line-height: 14px;
  color: #616070;
}
```
[рекоммендации](http://codeguide.academy/html-css.html#css-order)